app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb

```diff
@@ -1,25 +1,26 @@
 <%= content_tag(:div,
-    id: object.id,
-    data: object.data,
-    class: object.classname) do %>
-    <%= pb_rails("person", props: {
-        first_name: object.first_name,
-        last_name: object.last_name,
+  id: object.id,
+  data: object.data,
+  class: object.classname) do %>
+  <%= pb_rails("person", props: {
+      first_name: object.first_name,
+      last_name: object.last_name,
+  }) %>
+  <% object.valid_contacts.each do |contact| %>
+    <%= pb_rails("contact", props: {
+        contact_type: contact[:contact_type],
+        contact_value: contact[:contact_value],
+        contact_detail: contact[:contact_detail],
     }) %>
-    <% object.contacts.each do |contact| %>
+  <% end %>
+  <% unless object.wrong_contacts.empty? %>
+    <%= pb_rails("caption", props: { text: "wrong number" }) %>
```

Comment:
Looking at the screenshot (thanks for providing) I believe this caption needs some margin-top. Can you match it up with the mocks, and use the spacing variables to give it some separation? Let me know if you need help!

Reply:
I threw the caption kit here into a `div` with some `padding-top: 10px`. Is that an OK way to do that, @jasoncypret ?